### DISCUSSION TITLE
I don't know what to put

### DISCUSSION THREAD

**Discussion by CECHAMO81** [FunkinCrew/Funkin#2043] [Ideas]
It really is not to bother but it would be better if the original game engine was more flexible when creating a mod or adding a complement or edition of the same code and also being able to configure the game options to adapt it to slow or high-end PCs (because neither week 1 nor week 6 worked for me) (sorry for any translation errors)

**Comment by RevixDev** [comment]
I'm pretty sure that NinjaMuffin is still working on the Mods folder if I am correct.

**Comment by CECHAMO81** [comment]
I know that "maybe" is working on that folder but it would be better if someone who knows help with this issue to speed up the game creation process since many are desperate for the next update and a small engine change would attract again to players and creators

**Comment by CECHAMO81** [comment]
also that I would like them to take into account the Latin American community xd because I come from there and an improvement would be to add translations or texts AND/or alternative audios by language
